sklearn/tests/test_base.py

```diff
@@ -436,6 +436,7 @@ def __init__(self, attribute_pickled=5):
 
     def __getstate__(self):
```

Comment:
Hi @jnothman thanks for your review!
I am a newbie here.
I think it may not be appropriate to use `super(...).__getstate__` because the next test case is `test_pickling_works_when_getstate_is_overwritten_in_the_child_class`.